MycroftAI/mycroft-core
# CommonIOT/acknowledge

## Description
Adds a function to MycroftSkill to acknowledge successful requests non-verbally. This is intended to be used in the commonIoT control skill, to acknowledge that a request was successfully passed to an appropriate skill.

Note that this also add a few key THING types for working with thermostats. I figured it would be easier to include that here than have a separate PR for the three lines in `common_iot_skill.py`, but am happy to split that out into it's own PR at your preference.

## How to test
Pull down the `improvement/acknowledge` branch of `skill-iot-control`, and run an IoT command (see  https://github.com/MycroftAI/skill-iot-control/pull/2, which will be merged after this PR goes through).

## Contributor license agreement signed?
CLA [x ] (Whether you have signed a [CLA - Contributor Licensing Agreement](https://mycroft.ai/cla/)
